doc/api.rst

```diff
@@ -57,10 +57,24 @@ contains the p-value and the test statistic and optionally additional informatio
 Testing for conditional independence among variables is a core part
 of many data analysis procedures.
 
-.. currentmodule:: pywhy_stats
+.. currentmodule:: pywhy_stats.independence
 .. autosummary::
    :toctree: generated/
-
+   
    fisherz
    kci
 
+
+(Conditional) Discrepancy Testing
```

Comment:
Perhaps Conditional K-Sample Testing?

I use discrepancy because I saw it in one paper, but I think (conditional) k-sample is more correct. I guess in our cases, we are only doing conditional 2-sample testing, since it's P_1(y|x) =? P_2(y|x)

Reply:
I changed it